packages/backend/src/api/controllers/status/StatusController.ts

```diff
@@ -89,28 +75,45 @@ export class StatusController {
   }
 }
 
+function isSynced(timestamp: UnixTime | undefined) {
+  const now = UnixTime.now().add(-1, 'hours').toStartOf('hour')
+  return !!timestamp && now.equals(timestamp)
+}
+
 const unixTimeToString = (date: UnixTime) => {
   return date.toDate().toString().slice(4, 21)
 }
 
 const SECONDS_PER_HOUR = 60 * 60
-const getSyncStatus = (date: UnixTime): string => {
+const getSyncStatus = (
```

Comment:
I presume this function will get removed once we have html pages for every kind of status?

Reply:
yes